contracts/src/LiquidityPoolV2.sol

```diff
@@ -90,10 +123,21 @@ contract LiquidityPoolV2 is ReentrancyGuardUpgradeable, AccessManagedUpgradeable
         emit LiquidityRemoved(msg.sender, tokenAAmount, tokenBAmount);
```

Comment:
adjustForDecimals (see last comment of the file to understand)

Can't comment the good line idk why but replace tokenAAmount and tokenBAmount in the transfers but the adjustedToken